jsonnetfile.json

```diff
@@ -1,13 +1,14 @@
 {
+  "version": 1,
   "dependencies": [
     {
-      "name": "kube-prometheus",
       "source": {
         "local": {
           "directory": "jsonnet/kube-prometheus"
         }
       },
       "version": ""
     }
-  ]
+  ],
+  "legacyImports": true
```

Comment:
hi, don't you consider this as breaking change, because the old version of jb bundler won't work with that?

Reply:
it's a breaking change in jb not here, we're just upgrading to latest tooling, which everyone should do